pages/index.js

```diff
@@ -49,6 +49,45 @@ export default function Home() {
 
     const myHeight = ((!isMobile) ? "calc(100vh - 170px)" : "calc(50vh - 85px)");
 
+    useEffect(() => {
+            if(moduleReady) { fetchData(); }
+      }, [moduleReady]);
```

Comment:
I think this should just be 
```suggestion
    useEffect(() => {
           fetchData(); 
      }, []);
```
I think it should not depend on `moduleReady`. If it depends on `moduleReady`, then I think it leads to two API calls. One API call would be as soon as the web page is opened in the browser. I think another API call would be made when module becomes ready (that is `moduleReady` is set/updated to `true`). The second API call seems redundant as the data obtained by the first API call and the data obtained by the second API call would be the same.

Reply:
Sure.